src/main/java/cm/ptks/craftflowers/flower/FlowerRegistry.java

```diff
@@ -25,6 +25,23 @@ public class FlowerRegistry {
         registerFlower(new Flower(Material.DEAD_BUSH, "§2Dead Bush"));
         registerFlower(new Flower(Material.GRASS, "§2Grass"));
         registerFlower(new Flower(Material.FERN, "§2Fern"));
+        registerFlower(new Flower(Material.NETHER_BRICK_FENCE, "§2Nether Brick Fence"));
+
+        registerFlower(new Flower(Material.DARK_OAK_FENCE, "§2Dark Oak Fence"));
+        registerFlower(new Flower(Material.SPRUCE_FENCE, "§2Spruce Fence"));
+        registerFlower(new Flower(Material.OAK_FENCE, "§2Oak Fence"));
+        registerFlower(new Flower(Material.BIRCH_FENCE, "§2Birch Fence"));
+        registerFlower(new Flower(Material.JUNGLE_FENCE, "§2Jungle Fence"));
+        registerFlower(new Flower(Material.ACACIA_FENCE, "§2Acacia Fence"));
+
+        if (isValidMaterial("WARPED_FENCE") /* 1.16+ */) {
```

Comment:
This is not required since 1.16 is our base version. So you dont need to check for 1.16